eras/alonzo/impl/testlib/Test/Cardano/Ledger/Alonzo/TreeDiff.hs

```diff
@@ -172,3 +176,17 @@ instance
         , ("pwcExUnits", toExpr pwcExUnits)
         , ("pwcCostModel", toExpr pwcCostModel)
         ]
+
+deriving instance Generic (TransactionScriptFailure era)
+
+instance
+  ( Era era
+  , ToExpr (TxCert era)
+  , ToExpr (ContextError era)
+  , ToExpr (PlutusScript era)
+  , ToExpr (PlutusPurpose AsIx era)
+  , ToExpr (PlutusPurpose AsItem era)
+  , ToExpr EvaluationError
+  , ToExpr ExBudget
```

Comment:
These two constraints are fully saturated. Which means they cannot be satisfied.